src/test/resources/assets/minecraft/textures/block/copper_ore.png.mcmeta

```diff
@@ -0,0 +1,9 @@
+{
+    "forge": {
+        "loader": "oretexgen:loader"
+    },
+    "oretexgen": {
+        "background": "minecraft:block/blackstone",
+        "shadow": true
```

Comment:
Is it intentional that specifying custom shadow parameters is not tested here?

Reply:
Sorta; the purpose of this new test was to test how the algorithm works on stuff other than stone - and, well, the default settings of the configuration are pretty close to optimal. But yeah, I'll make a test for the custom configuration stuff.

Reply:
I've added a new test for this